### ISSUE TITLE
Can't find Neerslag App in HACS

### ISSUE THREAD

**Issue by JanLankhorst** [aex351/home-assistant-neerslag-app#9]
Hi,

I can't seem to find Neerslag App in HACS Integrations...Am I doing something wrong? It is not the first HACS integration I am installing (nor the first add on in Homeassistant)...

Jan

**Comment by aex351** [comment]
Hi Jan, that is correct. The Neerslag App currently is marked and labeled as a 'custom' HACS integration. The Neerslag App is not part of the `default` HACS repository. This means that you as a user will need to manually add the Neerslag App to HACS.

You can do this by navigating to HACS in Home Assistant. Click on `integrations` and in the right top corner you will find a menu. In this menu click on `custom repositories` and add the Neerslag App GitHub URL. 

For future reference I will see if I can add the Neerslag App to the `default` HACS repository. There are still some open tasks that need to be completed before this can happen.
